phpstan.neon

```diff
@@ -1,4 +1,5 @@
-#parameters:
-#    ignoreErrors:
+parameters:
+    ignoreErrors:
+        - "#Call to an undefined static method SoapClient::__construct\\(\\)#"
```

Comment:
That's weird, it was a constructor right? http://us2.php.net/manual/en/soapclient.construct.php
Or should we refactor to call the SoapClient method?

Reply:
Yes, it has a constructor. This is a PHPStan bug.
That is why I added the PHPStan in the `ignoreErrors` section.

I should probably open an issue in the PHPStan repo regarding this.

Reply:
Maybe someone created an extension for it

Reply:
Ho, no, I just discovered it.

It is most definitely a bug in PHPStan. The soap extension is enabled and the error is still present.

I think Ondrej nailed the problem here: https://github.com/phpstan/phpstan/issues/618#issuecomment-347240188

I'm pretty sure the issue will be solved in the next PHPStan release. We can keep the `ignoreErrors` parameter and remove it when the bug is fixed in PHPStan.